lib/vscode/src/vs/workbench/contrib/extensions/browser/extensionsViewlet.ts

```diff
@@ -406,10 +406,44 @@ export class ExtensionsViewPaneContainer extends ViewPaneContainer implements IE
 		this.root = parent;
```

Comment:
I can't comment on this in github but the commit message here should be hard wrapped to 80 lines.
It's a common standard when using git.

Reply:
Ah...tbh I was trying to write more descriptive git commit messages after looking at some of yours. Didn't realize I had to wrap it myself. Is there a setting I can change in my git configuration to prevent this from happening in the future?

Reply:
I wrapped it using [this method](https://stackoverflow.com/a/3033455/3015595) but it would be great to configure that automatically for next time.

Reply:
Should be able to configure VS Code to wrap for you. In fact it should be the default. Are you committing via vim on the CLI or VS Code? For vim there's some settings you have to set to make it automatic but I don't remember off the top of my head.

Reply:
For this specific commit, it was via Vim in the VSCode terminal. No worries at all. Thanks!